posts/2020-08-27-Rust-1.46.0.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Announcing Rust 1.46.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.46.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.46.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.46.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1460-2020-08-27
+
+## What's in 1.46.0 stable
+
+This release is on the smaller side, with a number of improvements to `const
```

Comment:
const fn improvements are really big to me! It's not a small release.

Reply:
Yes, as mentioned above, I'm changing this. I did not realize how much power this gave folks; and it's a pretty small *number* of things, so I mis-judged!